Exporting report with dockerfile creates panics

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

The experimental reports panics with the dockerfile resource

### Expected Behavior

_No response_

### Steps To Reproduce

```
go run main.go diff --config e2e/updatecli.d/success.d/dockerfile/dockerfile.yaml --experimental
```

The experimental export panics
```bash
REPORT - EXPERIMENTAL
======================

panic: duplicated key 'value' in struct dockerfile.Spec [recovered]
	panic: duplicated key 'value' in struct dockerfile.Spec

goroutine 1 [running]:
gopkg.in/yaml%2ev3.handleErr(0x14000ab7140)
	gopkg.in/yaml.v3@v3.0.1/yaml.go:294 +0x7c
panic({0x1046614c0?, 0x14000e887f0?})
	runtime/panic.go:792 +0x124
gopkg.in/yaml%2ev3.(*encoder).structv(0x14000966808, {0x0, 0x0}, {0x104b404e0?, 0x14000c12f00?, 0x2503c?})
	gopkg.in/yaml.v3@v3.0.1/encode.go:217 +0xd8
gopkg.in/yaml%2ev3.(*encoder).marshal(0x14000966808, {0x0, 0x0}, {0x104b404e0?, 0x14000c12f00?, 0x101164de4?})
	gopkg.in/yaml.v3@v3.0.1/encode.go:168 +0x6e4
gopkg.in/yaml%2ev3.(*encoder).marshal(0x14000966808, {0x0, 0x0}, {0x104600080?, 0x14000b127d0?, 0x0?})
	gopkg.in/yaml.v3@v3.0.1/encode.go:162 +0x614
gopkg.in/yaml%2ev3.(*encoder).structv.func1()
	gopkg.in/yaml.v3@v3.0.1/encode.go:235 +0x198
gopkg.in/yaml%2ev3.(*encoder).mappingv(0x14000966808, {0x0?, 0x0}, 0x14000ab5d58)
	gopkg.in/yaml.v3@v3.0.1/encode.go:265 +0xfc
gopkg.in/yaml%2ev3.(*encoder).structv(0x14000966808, {0x0, 0x0}, {0x104c57fa0?, 0x14000b12780?, 0x20101?})
	gopkg.in/yaml.v3@v3.0.1/encode.go:219 +0xc0
gopkg.in/yaml%2ev3.(*encoder).marshal(0x14000966808, {0x0, 0x0}, {0x104c57fa0?, 0x14000b12780?, 0x101184cf4?})
	gopkg.in/yaml.v3@v3.0.1/encode.go:168 +0x6e4
gopkg.in/yaml%2ev3.(*encoder).marshal(0x14000966808, {0x0, 0x0}, {0x104600080?, 0x14000b259d0?, 0x0?})
	gopkg.in/yaml.v3@v3.0.1/encode.go:162 +0x614
gopkg.in/yaml%2ev3.(*encoder).structv.func1()
	gopkg.in/yaml.v3@v3.0.1/encode.go:235 +0x198
gopkg.in/yaml%2ev3.(*encoder).mappingv(0x14000966808, {0x0?, 0x0}, 0x14000ab6378)
	gopkg.in/yaml.v3@v3.0.1/encode.go:265 +0xfc
gopkg.in/yaml%2ev3.(*encoder).structv(0x14000966808, {0x0, 0x0}, {0x104d6fb40?, 0x14000b258c8?, 0x2b3940?})
	gopkg.in/yaml.v3@v3.0.1/encode.go:219 +0xc0
gopkg.in/yaml%2ev3.(*encoder).marshal(0x14000966808, {0x0, 0x0}, {0x104d6fb40?, 0x14000b258c8?, 0x0?})
	gopkg.in/yaml.v3@v3.0.1/encode.go:168 +0x6e4
gopkg.in/yaml%2ev3.(*encoder).marshal(0x14000966808, {0x0, 0x0}, {0x10471b920?, 0x14000b258c8?, 0x98?})
	gopkg.in/yaml.v3@v3.0.1/encode.go:166 +0x684
gopkg.in/yaml%2ev3.(*encoder).marshal.(*encoder).mapv.func1()
	gopkg.in/yaml.v3@v3.0.1/encode.go:192 +0xd4
gopkg.in/yaml%2ev3.(*encoder).mappingv(0x14000966808, {0x0?, 0x0}, 0x14000ab6a10)
	gopkg.in/yaml.v3@v3.0.1/encode.go:265 +0xfc
gopkg.in/yaml%2ev3.(*encoder).mapv(...)
	gopkg.in/yaml.v3@v3.0.1/encode.go:187
gopkg.in/yaml%2ev3.(*encoder).marshal(0x14000966808, {0x0, 0x0}, {0x1047bdf00?, 0x14000460890?, 0x0?})
	gopkg.in/yaml.v3@v3.0.1/encode.go:164 +0x654
gopkg.in/yaml%2ev3.(*encoder).structv.func1()
	gopkg.in/yaml.v3@v3.0.1/encode.go:235 +0x198
gopkg.in/yaml%2ev3.(*encoder).mappingv(0x14000966808, {0x0?, 0x0}, 0x14000ab6d18)
	gopkg.in/yaml.v3@v3.0.1/encode.go:265 +0xfc
gopkg.in/yaml%2ev3.(*encoder).structv(0x14000966808, {0x0, 0x0}, {0x104ccfba0?, 0x14000460828?, 0x2b3d80?})
	gopkg.in/yaml.v3@v3.0.1/encode.go:219 +0xc0
gopkg.in/yaml%2ev3.(*encoder).marshal(0x14000966808, {0x0, 0x0}, {0x104ccfba0?, 0x14000460828?, 0x0?})
	gopkg.in/yaml.v3@v3.0.1/encode.go:168 +0x6e4
gopkg.in/yaml%2ev3.(*encoder).marshal(0x14000966808, {0x0, 0x0}, {0x1047ffee0?, 0x14000460828?, 0x1011834f8?})
	gopkg.in/yaml.v3@v3.0.1/encode.go:166 +0x684
gopkg.in/yaml%2ev3.(*encoder).marshalDoc(0x14000966808, {0x0, 0x0}, {0x1047ffee0?, 0x14000460828?, 0x1400000000e?})
	gopkg.in/yaml.v3@v3.0.1/encode.go:105 +0x11c
gopkg.in/yaml%2ev3.Marshal({0x1047ffee0, 0x14000460828})
	gopkg.in/yaml.v3@v3.0.1/yaml.go:222 +0x268
github.com/updatecli/updatecli/pkg/core/reports.(*Report).ExportToYAML(0x14000460828, {0x0?, 0x103a0442e?}, 0x4?)
	github.com/updatecli/updatecli/pkg/core/reports/export.go:45 +0x194
github.com/updatecli/updatecli/pkg/core/engine.(*Engine).exportReportToYAML(0x1073273c0, 0x0)
	github.com/updatecli/updatecli/pkg/core/engine/exportReport.go:24 +0x188
github.com/updatecli/updatecli/pkg/core/engine.(*Engine).Run(0x1073273c0)
	github.com/updatecli/updatecli/pkg/core/engine/run.go:40 +0x30c
github.com/updatecli/updatecli/cmd.run({0x103a0434a?, 0x0?})
	github.com/updatecli/updatecli/cmd/root.go:132 +0x4f0
github.com/updatecli/updatecli/cmd.init.func9(0x1072d5400?, {0x14000aa87b0?, 0x0?, 0x3?})
	github.com/updatecli/updatecli/cmd/diff.go:38 +0x228
github.com/spf13/cobra.(*Command).execute(0x1072d5400, {0x14000aa8750, 0x3, 0x3})
	github.com/spf13/cobra@v1.9.1/command.go:1019 +0x82c
github.com/spf13/cobra.(*Command).ExecuteC(0x1072d3e00)
	github.com/spf13/cobra@v1.9.1/command.go:1148 +0x384
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.9.1/command.go:1071
github.com/updatecli/updatecli/cmd.Execute()
	github.com/updatecli/updatecli/cmd/root.go:50 +0x68
main.main()
	github.com/updatecli/updatecli/main.go:8 +0x1c
```


### Environment

```markdown
- OS:
- updatecli:
```

### Pipeline Graph

```mermaid
```


### Anything else?

_No response_

--- a/pkg/plugins/resources/dockerfile/main.go
+++ b/pkg/plugins/resources/dockerfile/main.go
@@ -27,7 +27,7 @@ type Spec struct {
 	// - If not defined, the last stage will be considered
 	// For Condition and Targets:
 	// - If not defined, all stages will be considered
-	Stage string `yaml:"value,omitempty"`
+	Stage string `yaml:"stage,omitempty"`
 }
 
 // Dockerfile defines a resource of kind "dockerfile"

